tools/build.sh

```diff
@@ -86,15 +86,15 @@ _generate() {
         --java_out=$PROTOC_OUT \
         $proto_src
 
-    # echo "# - validation tests"
-    # $PROTOC_PATH \
-    #     --plugin=protoc-gen-java-snowpea=protoc-gen-java-snowpea/startup-script.sh \
-    #     --java-snowpea_out $TEST_GEN_OUT \
-    #     --java-snowpea_opt "MessagePublishedData,LogEntryData" \
-    #     -I $PROTOBUF_SRC \
-    #     -I $THIRDPARTY \
-    #     --experimental_allow_proto3_optional \
-    #     $proto_src
+    echo "# - validation tests"
+    $PROTOC_PATH \
```

Comment:
Have you considered using the [Xolstice protobuf-maven-plugin](https://www.xolstice.org/protobuf-maven-plugin/)?

Reply:
No, we are trying to minimize any additional dependencies and rely on tools that have long term maintenance plans.